src/Miunie.Core/Services/TimeService.cs

```diff
@@ -45,6 +47,25 @@ public async Task OutputCurrentTimeComparedToInputForUserAsync(MiunieUser reques
             await _messages.SendMessageAsync(channel, PhraseKey.TIME_USERTIME_FROM_LOCAL, formattedRequestTime, user.Name, formattedOtherUserTime);
         }
 
+        public async Task OutputFutureTimeForUserAsync(MiunieUser user, int units, string timeframe, MiunieChannel channel)
+        {
+            var timeFromLocal = _timeManipulator.GetTimeSpanFromString(timeframe, units);           
+
+            if (timeFromLocal is null)
+            {
+                await _messages.SendMessageAsync(channel, PhraseKey.TIME_USERTIME_IN_FUTURE_UNPARSABLE, units.ToString(), timeframe);
+                return;
+            }
+
+            var usersOffset = user.UtcTimeOffset ?? new TimeSpan();
+            var usersLocalTime = _dateTime.UtcNow + usersOffset;
+            var usersFutureTime = usersLocalTime + timeFromLocal;
+
+            var formattedUsersTime = usersFutureTime?.ToShortTimeString();
```

Comment:
I think we should let the phrase format the time. Similar to how it is in the `time for` command.
So just passing the DateTime into the phrase and having it be formatted kinda like `{0:HH:mm}` or whatever format is needed. 🙂 

Should be an easy enough fix. 😊